### PR TITLE
Revert ARK rebase PR #1248

### PR DESCRIPTION
**Pull request by weiweishi**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -34,5 +34,4 @@
   config.assets.debug = true
 
   BetterErrors::Middleware.allow_ip! '192.168.0.1/16'
-  Rails.application.routes.default_url_options[:host] = 'hydranorthdev.library.ualberta.ca'
```
Comment (mbarnett) [inline config/environments/development.rb]: Was this mean to be deleted with the ARK stuff? Seems unrelated.

File: config/environments/test.rb [hunk 2]
```diff
@@ -44,5 +44,4 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.action_mailer.default_url_options = { :host => 'localhost' }
-  Rails.application.routes.default_url_options[:host] = 'hydranorthdev.library.ualberta.ca'
```
Comment (mbarnett) [inline config/environments/test.rb]: Ditto this.

**Comment by mbarnett** [comment]
Er, not better_errors. Hosts for URLs.

**Comment by pbinkley** [comment]
We intend to remove these definitions and implement them in the Ansible playbook (from Tricia's Sufia 7 branch).
